client/src/components/form-builder/form-builder.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Box, Button, Container, Divider, Typography } from '@mui/material';
+import { FormProvider, useFieldArray, useForm } from 'react-hook-form';
+import { FormTemplateSchema, validateWithLatest } from '@konomi.ai/c-form';
+import Spacer from '../spacer/Spacer';
+import { DEFAULT_SCHEMA_VAL, DEFAULT_SECTION_VAL } from './const';
+import { FormSection } from './form-section';
+
+export const FormBuilder = () => {
+  const form = useForm<FormTemplateSchema>({
+    defaultValues: DEFAULT_SCHEMA_VAL,
+  });
+  const { fields, append, remove } = useFieldArray({
+    control: form.control,
+    name: 'sections',
+  });
+  const onSubmit = (data: FormTemplateSchema) => {
+    const valid = validateWithLatest(data);
+    if (!valid) {
+      console.log(validateWithLatest.errors);
+    }
+  };
+
+  return (
+    <Container>
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+      <FormProvider {...form}>
+        {fields.map((f, i) => (
```

Comment:
readability comment here. I know `fields` is the value given by `useFormArray` but since this is for the `FormSection` component we could de-structuring `fields` as `sections` instead

in fact it might not be a bad for the other times `fields` from `useFormArray` just to know at a glance what we are mapping out